src/main/java/com/github/susom/dbgoodies/etl/Etl.java

```diff
@@ -80,6 +82,11 @@ public SaveAsAvro asAvro(String path, String schemaName, String tableName) {
       return new SaveAsAvro(path, schemaName, tableName, select);
     }
 
+    @CheckReturnValue
+    public SaveAsAvro asAvro(String path, String schemaName, String tableName, int fetchSize, CodecFactory codec) {
+      return new SaveAsAvro(path, schemaName, tableName, select).withCodec(codec).fetchSize(fetchSize);
+    }
+
```

Comment:
Doesn't this sort of violate the builder style (param ==> mandatory; method ==> optional with default)? Why not chain `saveQuery().asAvro().fetchSize().withCodec()` rather than including extra params in `asAvro()`?

Reply:
Yes it totally does, but fetchSize() already wasn't public, so I made withCodec() also not public in case there was a reason for that -- so I glued it together. If there isn't any reason for them not being public then let's do that.

Reply:
Oh, I hadn't noticed that. I think the fact that it isn't already public is an oversight on my part.